### PR TITLE
Feature/homepage display pokemons

### PR DESCRIPTION
**Pull request by raymond-le-goldenowl**
**Task**
- Display pokemon without pagination

**Descriptions**
- Get pokemons with api link and create a Card to display to screen

**Screenshots**
Screenshot from 2021-12-30 12-17-26

### CODE REVIEW

File: src/pages/home-page.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+
+  const fetchPokemonsPaginateData = useCallback((offset = 0, limit = 20) => {
+    const url = `https://pokeapi.co/api/v2/pokemon?${offset}=1&limit=${limit}`
+    return axios.get(url).then(response => response.data)
+  }, [])
+
+  useEffect(() => {
+    fetchPokemonsPaginateData().then(pokemonsPaginateData => {
+      setPokemonsPaginateData(pokemonsPaginateData?.results)
+    })
```
Comment (jun-nguyen-goldenowl) [inline src/pages/home-page.js:16]: what will happened if api fetch is failed?

File: src/pages/home-page.js [hunk 2]
```diff
@@ -23,125 +37,13 @@ export default function HomePage() {
...
-              </p>
-              <a href="#!" className="btn btn-purple">
-                Go somewhere
-              </a>
-            </div>
-          </div>
-        </div>
+        {pokemonsPaginateData.map((pokemon, index) => (
+          <CardPokemon
+            key={index}
```
Comment (jun-nguyen-goldenowl) [inline src/pages/home-page.js]: DO NOT use array's item index as element key.

**Review by jun-nguyen-goldenowl — APPROVED** [review]

File: src/pages/home-page.js (hunk 1 above)
Comment (raymond-le-goldenowl) [reply src/pages/home-page.js:16]: App still run but not data showing

File: src/pages/home-page.js (hunk 2 above)
Comment (raymond-le-goldenowl) [reply src/pages/home-page.js]: Fixed

**Pull request merged**